app/kumactl/cmd/export/export.go

```diff
@@ -180,3 +180,9 @@ func resourcesTypesToDump(ctx context.Context, ectx *exportContext) ([]model.Res
 	}
 	return resTypes, nil
 }
+
+func includeInFederationProfile(res api_common.ResourceTypeDescription) bool {
+	return res.IncludeInFederation && // base decision on `IncludeInFederation` field
+		(res.Policy == nil || (res.Policy != nil && !res.Policy.IsTargetRef)) && // do not include new policies (q: should this just be reflected in IncludeInFederation?)
```

Comment:
mostly question to @jakubdyszkiewicz

Reply:
I don't think it should. If you set `includeInFederation:true` to target ref policies then you would need to filter this out in profileFederation